src/components/PupilViews/PupilExperience/PupilExperience.view.tsx

```diff
@@ -158,6 +158,11 @@ export const PupilExperienceView = ({
   const router = useRouter();
   const availableSections = pickAvailableSectionsForLesson(lessonContent);
 
+  const isSensitive: boolean =
+    typeof lessonContent.deprecatedFields?.isSensitive === "boolean"
+      ? lessonContent.deprecatedFields.isSensitive
+      : false;
```

Comment:
Unfortunately the isSensitive property was being interpreted as a possible object, and this was the only way around that for the moment.

Reply:
If we can't fix this. Maybe add a big `NOTE` comment on that because that's super confusing, future me will thank you  🙂